sdk/firstload/src/index.tsx

```diff
@@ -217,30 +217,32 @@ export const H: HighlightPublicInterface = {
 			)
 			const highlightUrl = highlight_obj?.getCurrentSessionURL()
 
-			if (window.mixpanel?.track) {
-				window.mixpanel.track(event, {
-					...metadata,
-					highlightSessionURL: highlightUrl,
-				})
+			if (!H.options?.integrations?.mixpanel?.disabled) {
+				if (window.mixpanel?.track) {
+					window.mixpanel.track(event, {
+						...metadata,
+						highlightSessionURL: highlightUrl,
+					})
+				} else {
+					console.warn(
+						"Mixpanel not loaded, but Highlight is configured to use it. This is usually caused by Mixpanel being blocked by the user's browser.",
+					)
+				}
 			}
 
-			if (
-				!!H.options?.integrations?.mixpanel?.projectToken &&
-				!window?.mixpanel?.track
-			) {
-				console.warn(
-					"Mixpanel not loaded, but Highlight is configured to use it. This is usually caused by Mixpanel being blocked by the user's browser.",
-				)
+			if (!H.options?.integrations?.amplitude?.disabled) {
+				if (window.amplitude?.getInstance) {
+					window.amplitude.getInstance().logEvent(event, {
+						...metadata,
+						highlightSessionURL: highlightUrl,
+					})
+				}
 			}
 
-			if (window.amplitude?.getInstance) {
-				window.amplitude.getInstance().logEvent(event, {
-					...metadata,
-					highlightSessionURL: highlightUrl,
-				})
-			}
-			if (window.Intercom) {
-				window.Intercom('trackEvent', event, metadata)
+			if (!H.options?.integrations?.intercom?.disabled) {
```

Comment:
nit: same if as 233?

Reply:
This one is for `intercom` instead of `amplitude`. Is that incorrect? Or do you see a refactor that could improve this?

Reply:
oh my bad!